### ISSUE TITLE
Some Top site tiles are missing favicons

### ISSUE THREAD

**Issue by btlechowski** [brave/browser-laptop#13728]
### Steps to Reproduce
  1. Clean install
  2. Open onet.pl
  3. Open a new tab

**Actual result:**
Instead of favicons, there are letters (on screenshot below O)
![image](https://user-images.githubusercontent.com/34715963/38328923-7aca5088-384c-11e8-996a-32e6fa343137.png)

When I click on the O tile, the favicon will load.

**Expected result:**
Favicons are immediately shown

### Brave Version
Reproduced on Win7 and Mac
```
Brave: 0.22.103 
V8: 6.5.254.41 
rev: 2324f5563110b178304d30b81f3cfae965ae4a03 
Muon: 6.0.0 
OS Release: 6.1.7601 
Update Channel: Beta 
OS Architecture: x64 
OS Platform: Microsoft Windows 
Node.js: 7.9.0 
Brave Sync: v1.4.2 
libchromiumcontent: 65.0.3325.181

```

**Comment by petemill** [comment]
I think this is fixed with 0.22.106, please let me know if otherwise

**Comment by Tomatoshadow2** [comment]
@petemill Is it still possible for what @btlechowski is describing to show up like that for any site, as I just downloaded 22.106 and one of the tiles are like that? 

Thanks.

**Comment by petemill** [comment]
Should be fixed with 0.22.108

**Comment by LaurenWags** [comment]
Site listed in STR works, but other sites (twitter and github) are still displaying a letter instead of favicon. Tested with 0.22.109:
![13728](https://user-images.githubusercontent.com/28145373/38639654-e4392abe-3d9f-11e8-9543-cc3deded6d02.gif)

**Comment by LaurenWags** [comment]
It looks like instead of displaying the favicons it's displaying the first letter of the tab title. In the below image I opened 0.22.109 with a clean profile then closed the about:welcome page. From the new tab page, I cmd+clicked on each of the default top site tiles. After a few seconds, the favicons changed to the letters (only the pinned twitter tab retained the favicon):
![screen shot 2018-04-11 at 5 45 24 pm](https://user-images.githubusercontent.com/28145373/38645060-ba727daa-3db0-11e8-8129-1fd6427c7122.png)

**Comment by petemill** [comment]
Found the bug and fixed it. Should be fixed with 0.22.701
